libfreefare/ntag21x.c

```diff
@@ -137,14 +141,14 @@ _ntag21x_tag_reuse(FreefareTag old_tag)
 	tag->device = old_tag->device;
 	tag->info = old_tag->info;
 	tag->active = 0;
-	NTAG_21x(tag)->subtype = NTAG_213; // Set tag subtype
-	NTAG_21x(tag)->vendor_id = 0x04;
-	NTAG_21x(tag)->product_type = 0x04;
-	NTAG_21x(tag)->product_subtype = 0x02;
-	NTAG_21x(tag)->major_product_version = 0x01;
-	NTAG_21x(tag)->minor_product_version = 0x00;
-	NTAG_21x(tag)->storage_size = 0x0f;
-	NTAG_21x(tag)->protocol_type = 0x03;
+	NTAG_21x(tag)->subtype = NTAG_21x(old_tag)->subtype; // Get from old tag
```

Comment:
This seems indeed better.  Not sure the comment on this line and the next ones add any value though 😉 Can you please remove them?

Reply:
I will :+1: